DigitalExcellence/dex-backend
# Backend cleanup

## Description

API, Service and Repository layer are refactored:
- Added missing summaries
- Removed redundant 'usings'
- Added missing license headers
- Removed redundant code
- Renamed projects so projects in solution explorer are ascending by default

## Type of change


## Checklist


## Steps to Test or Reproduce

Only redundant code was removed.
Run code and check if everything still works. See Postman tests.
